### DISCUSSION TITLE
input.http source can't changed

### DISCUSSION THREAD

**Discussion by xamke** [savonet/liquidsoap#1965] [Q&A]
**Describe the bug**
input.http source url can't change

**Expected behavior**

**Version details**
 - OS: Ubuntu`

http_src = input.http(id="http_src", max=20.,buffer=10., "http://www.xxxx.com:8021/stream")
http_src = audio_to_stereo(http_src)

#radio fallback
radio = fallback(track_sensitive = false, transitions=[transition,transition, transition],[http_src, fallback_src, security])

list
http_src : input.http
alsa_out(default) : output.alsa
var : interactive variables
fallback : playlist

telnet result
------------------------------------
http_src.url  http://www.new.com:8022/stream
http_src.stop
http_src.start
http_src.status
connected http://www.xxxx.com:8021/stream" <== still not changed.

How can I change input.http source url via telnet or other

**Comment by toots** [comment]
What version of liquidsoap are you using?

**Reply by xamke** [reply]
1.4.3 version

**Comment by toots** [comment]
I'd suggest to try with `2.0.x`. The `url` parameter there is a string getter, i.e. you can pass a callback `() -> string` and change the url.
